src/Lecture1.hs

```diff
@@ -119,4 +141,17 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String 
+lowerAndGreater n list = (show n) ++ " is greater than " ++ (show (findGreater n list)) ++ " elements and lower than " ++ (show (findLower n list)) ++ " elements"
+    where 
+        findLower :: Int -> [Int] -> Int
+        findLower _ [] = 0
+        findLower n (x:xs)
+            | x > n     = 1 + findLower n xs
+            | otherwise = findLower n xs
+
+        findGreater :: Int -> [Int] -> Int
+        findGreater _ [] = 0
+        findGreater n (x:xs)
```

Comment:
Your implementation is absolutely correct 👏🏻 
But it traverses the list twice. Generally, it's better to write functions that traverse lists only once. So they will consume each element only once and the composition of such functions would have nice streaming properties.